src/components/cc-select/cc-select.stories.js

```diff
@@ -163,7 +174,7 @@ export const disabled = makeStory(conf, {
   items: [
     {
       label: 'Favourite artist',
-      placeholder: '-- Select an artist --',
+      value: 'LENNON',
```

Comment:
it did not make much sense to have a placeholder saying "Select an artist" in a disabled select element.